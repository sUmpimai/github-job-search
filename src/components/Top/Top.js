import * as React from 'react';

const Top = () => {
    return (
       <header>
         <div className="navbar navbar-dark bg-dark shadow-sm">
          <div className="container">
            <strong className="navbar-brand d-flex align-items-center">Github Job Search</strong>
          </div>
         </div>
       </header>
    )
}

export default Top;