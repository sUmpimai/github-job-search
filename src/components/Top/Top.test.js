import * as React from 'react';
import { shallow } from 'enzyme'
import Top from './Top';

describe('Top Panel Tests', () => {
  let component;

  beforeEach(() => {
    component = shallow(<Top/>);
  })

  it('should render correctly', () => {
    expect(component).toMatchSnapshot();
  })

  it('should render logo text', () => {
    expect(component.find("strong").text()).toEqual("Github Job Search");
  })
})