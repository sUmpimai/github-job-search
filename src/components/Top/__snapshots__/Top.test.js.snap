// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Top Panel Tests should render correctly 1`] = `ShallowWrapper {}`;
